travis/before_install.sh

```diff
@@ -18,3 +18,11 @@ if [ -z "$TRAVIS_TAG" ]; then
 else
     echo "Travis docker cache disabled for tools/build on tag: $TRAVIS_TAG"
 fi
+
```

Comment:
Keep this change (which won't affect anything if OSX is not in the build matrix)